test/lib/flat_mutation_reader_assertions.hh

```diff
@@ -554,6 +577,16 @@ private:
             return {rt.position(), {rt.tombstone().timestamp, {}}};
         }
     }
+    void reset_rtc() {
+        _rtc.set_tombstone({});
+    }
+    void apply_rtc(const range_tombstone_change& rtc) {
+        if (!_rtc.tombstone()) {
+            _rtc = rtc;
+        } else {
+            _rtc.set_tombstone(rtc.tombstone());
```

Comment:
This is wrong if you intend to use `_rtc` position.
Why not keep only the mos recent tombstone as `tombstone _current_range_tombstone`, and
simply assign here `_current_range_tombstone = rtc.tombstone()` unconditionally?

Reply:
It's indeed enough to just keep a `tombstone` (no need to track the position)